sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2372,6 +2377,10 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setExtendedErrorInfo(true).build();
     }
 
+    public Write<T> withAllowTruncatedTimestamps() {
```

Comment:
Please add a docstring for this

Reply:
@TheNeuralBit Added the comments, please take a look.